### PR TITLE
PMM-6876 Operators list in response.

### PR DESCRIPTION
**Pull request by BupycHuk**
[PMM-6876](https://jira.percona.com/browse/PMM-6876)

- [ ] Build: https://github.com/Percona-Lab/pmm-submodules/pull/1302

- [ ] Links to other linked pull requests (optional).

- https://github.com/percona/pmm/pull/637
- https://github.com/percona-platform/dbaas-controller/pull/106
- https://github.com/percona-platform/dbaas-api/pull/77
- https://github.com/percona/pmm-managed/pull/573

### CODE REVIEW

File: service/k8sclient/k8client.go [hunk 1]
```diff
@@ -734,3 +751,35 @@ func (c *K8Client) volumeSpec(diskSize string) *common.VolumeSpec {
...
+	output, err := c.kubeCtl.Run(ctx, []string{"api-versions"}, "")
+	if err != nil {
+		return nil, errors.Wrap(err, "can't get api versions list")
+	}
+
+	apiVersions := strings.Split(string(output), "\n")
+
+	return &Operators{
+		Xtradb: c.checkOperatorStatus(apiVersions, pxcAPIVersion),
+		Psmdb:  c.checkOperatorStatus(apiVersions, psmdbAPIVersion),
```
Comment (askomorokhov) [inline service/k8sclient/k8client.go:765]: Looks like we have a collision in naming:
`Xtradb: c.checkOperatorStatus(apiVersions, pxcAPIVersion),`
here XtraDB vs PXC.

File: service/k8sclient/k8client.go [hunk 2]
```diff
@@ -734,3 +751,35 @@ func (c *K8Client) volumeSpec(diskSize string) *common.VolumeSpec {
...
+	}, nil
+}
+
+func (c *K8Client) checkOperatorStatus(installedVersions []string, expectedApiVersion string) (operator OperatorStatus) {
+	parts := strings.Split(expectedApiVersion, "/")
+	apiNamespace := parts[0]
```
Comment (askomorokhov) [inline service/k8sclient/k8client.go:771]:
```suggestion
	apiNamespace := strings.Split(expectedApiVersion, "/")[0]
```

**Review by askomorokhov — APPROVED** [review]
Overall looks good, but we have a collision in naming: PXC vs XtraDB

**Review by JiriCtvrtka — APPROVED** [review]

File: service/k8sclient/k8client.go (hunk 1 above)
Comment (BupycHuk) [reply service/k8sclient/k8client.go:765]: Not only here, I think it's good point to discuss on backend call. Will add it to agenda

**Comment by AlekSi** [comment]
@BupycHuk Please resolve merge conflicts

File: .golangci-required.yml [hunk 3]
```diff
@@ -10,6 +10,13 @@ linters-settings:
       - errors
       # use "github.com/golang/protobuf/proto" instead
       - github.com/gogo/protobuf/proto
+      # Use service/k8sclient/internal/... instead
+      - k8s.io/api/core/v1
+      - k8s.io/apimachinery/pkg/api/resource
+      - k8s.io/apimachinery/pkg/apis/meta/v1
+      - k8s.io/apimachinery/pkg/apis/runtime
+      - k8s.io/apimachinery/pkg/apis/types
+      - k8s.io/apimachinery/pkg/apis/util/intstr
```
Comment (AlekSi) [inline .golangci-required.yml:19]: Those changes look like a result of bad merge – they should not be in that PR

**Review by AlekSi — CHANGES_REQUESTED** [review]

File: .golangci-required.yml (hunk 3 above)
Comment (BupycHuk) [reply .golangci-required.yml]: Fixed merge conflicts and they lost.

**Pull request merged**